bin/dx-unpack

```diff
@@ -38,7 +38,7 @@ while [[ $# > 0 ]]; do
         output="$2"
         shift 2
     else
-        A="$A $1"
+        A="$A \"$1\""
```

Comment:
I am not sure this is the right fix. The addition of the quotes seems to introduce another problem when the filename itself contains quotes:

```
$ echo "a" > \"foo\ bar\"
$ dx-unpack \"foo\ bar\"
>>> Unpacking foo to /some/path
 *   dx_helpers.sh, line 20: foo does not exist
```

I think the "eval set" below is already supposed to fix up any problems with quoting as a result of this first phase. So I suspect any problems are downstream of that... perhaps in the call to unpack the arguments are being improperly tokenized (again)? I wonder if another judicious use of "eval set" could fix that...